mtp_noms_ops/apps/security/views/object_detail.py

```diff
@@ -101,15 +100,7 @@ class SenderDetailView(SecurityDetailView):
     object_context_key = 'sender'
 
     def get_title_for_object(self, detail_object):
-        try:
-            return detail_object['bank_transfer_details'][0]['sender_name']
-        except (KeyError, IndexError):
-            pass
-        try:
-            return detail_object['debit_card_details'][0]['cardholder_names'][0]
-        except (KeyError, IndexError):
-            pass
-        return _('Unknown sender')
+        return sender_profile_name(detail_object) or _('Unknown sender')
```

Comment:
You have a similar line in `security.forms.object_list` so maybe you could reuse some logic?